networking/src/Models/ITeamAnswer.ts

```diff
@@ -1,11 +1,12 @@
 import { ConfidenceLevel } from "../AWSMobileApi";
+import { ITeamAnswerContent } from "./AnswerClasses";
 
 export interface ITeamAnswer {
     id: string,
     questionId: number
     isChosen: boolean
     text: string
```

Comment:
I've been leaving it in until the backend updates are fully migrated. It's used throughout the codebase so it will be a breaking change to remove it. I'd like to get the multitable stuff sorted out and then pull it out at the end. Is that alright with you?

Reply:
Yeah definitely